misk-jdbc-testing/src/main/kotlin/misk/jdbc/MySqlScaleSafetyChecks.kt

```diff
@@ -1,13 +1,13 @@
 package misk.jdbc
 
 import com.zaxxer.hikari.util.DriverDataSource
-import misk.environment.Environment
 import net.ttddyy.dsproxy.proxy.ProxyConfig
 import net.ttddyy.dsproxy.support.ProxyDataSource
+import wisp.deployment.TESTING
 import java.sql.Connection
 import java.sql.SQLException
 import java.sql.Timestamp
-import java.util.Properties
+import java.util.*
```

Comment:
IntelliJ is fighting us?

Reply:
In a way, yes - I had to modify a setting after I reinstalled, these * changes are before I did that.  Will fix.